src/omerocrate/uploader.py

```diff
@@ -36,6 +139,8 @@ class OmeroUploader(BaseModel, arbitrary_types_allowed=True):
     Transfer method, which determines how images are sent to OMERO.
     `ln_s` is "in-place" importing, but it requires that this process has acess to both the image and permissions to write to the OMERO server.
     """
+    segmentation_uploader: SegmentationUploader
+    "SegmentationUploader instance to handle segmentation uploads."
```

Comment:
If the user has no segmentations, and doesn't want to have to choose an uploader, should we let this be `None`?

Reply:
Yes, seems reasonable. I've implemented this.